### ISSUE TITLE
Content notifications feature is visible but cannot be configured in Heartcore.

### ISSUE THREAD

**Issue by RyuLindow** [umbraco/Umbraco.Cloud.Issues#549]
### Issue description

This - https://our.umbraco.com/documentation/tutorials/Editors-Manual/Tips-and-Tricks/Notifications/ - cannot be configured in Heartcore since you need to add SMTP settings.

The docs state the following:
![image](https://user-images.githubusercontent.com/56826332/154529024-90e75d02-1109-47dd-8a11-7d6d17052eb3.png)

Yet the feature is visible. It's a bit confusing, so the feature should either be disabled or there should be an option to add your own SMTP settings :)

**Comment by VladProkopovich** [comment]
@sitereactor Any thoughts about this one? It's a crucial part of Umbraco functionality that's missing and the fact that it's not working on Heartcore isn't mentioned anywhere

**Comment by VladProkopovich** [comment]
@RyuLindow @sitereactor 
Can you please help escalate this one or at least guide where should we seek answers from? A huge part of the functionality is missing without any notice or workarounds and there is no communication at all here or with support.

**Comment by sitereactor** [comment]
The fact that Content Approvals / notifications is visable in Heartcore is actually a mistake, because we don't have an email service in place. So its unfortunate that it appears as a feature when really it should not be.
We have been talking internally about what we should do, as the obvious choice would be to disable it for all. But that wouldn't really be fair - even though its removing a currently unsupported feature.

Right now we are investigating if we can provide a webhook as an alternative.
So I apologize for the inconvincing, but we are trying to work out a solution.

**Comment by sitereactor** [comment]
No, we don't allow this type of customization on Umbraco Heartcore projects. So we'll need to solve it as part of the service, which is why its taking a bit longer to resolve.

**Comment by sitereactor** [comment]
Yes, I understand that and it's why I wrote that we want to find a fair solution to this inconvenience.

The user invite emails are delegated and handled as part of the service, so not sent from within Umbraco.

**Comment by sitereactor** [comment]
@VladProkopovich the work involved in getting this working in Heartcore has been added to our current sprint, which means that we are targetting an update roll-out for Tuesday the 29th.

**Comment by VladProkopovich** [comment]
Hey @sitereactor, thanks, that's amazing news. Does that mean that notifications in Heartcore will work the same way they are working on general Umbraco instances?

**Comment by sitereactor** [comment]
Content notifications, yes.
Please let me know if there are more/other notifications that fall under this umbrella.
Emails for Umbraco Forms is not part of this.

**Comment by VladProkopovich** [comment]
If I'm not mistaken, we are primarily interested in the content update notifications and the 'Save and send to approval' button. 
So if those will be working on the 29th - that's wonderful, thanks a lot for pushing this change.
